Sources/StreamVideo/WebRTC/v2/WebRTCStateAdapter.swift

```diff
@@ -74,7 +78,10 @@ actor WebRTCStateAdapter: ObservableObject {
     private let peerConnectionsDisposableBag = DisposableBag()
 
     /// Subject to handle participant updates.
-    private(set) lazy var participantsUpdateSubject = PassthroughSubject<[String: CallParticipant], Never>()
+    private let participantsUpdateSubject = PassthroughSubject<ParticipantsStorage, Never>()
+    private var participantsUpdatesCancellable: AnyCancellable?
+    private var interimParticipants = ParticipantsStorage()
```

Comment:
why exactly do we need interim participants?

Reply:
The interim participants were necessary when we were throttling the participants updates. That was because, the participants property was getting updated in a throttled manner, but in the meanwhile event were flowing. We needed then an interim value that will be updated with every event, without publishing updates to the outside world. 

Now that we moved the throttling to the UI, that property is unnecessary and now removed. Thank you for catching this :)